<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />
<meta name="viewport" content="width=device-width, initial-scale=1">



<title>Aquatic Plant eDNA with Nanopore</title>

<script src="lc_port_2021_files/header-attrs-2.7/header-attrs.js"></script>
<script src="lc_port_2021_files/accessible-code-block-0.0.1/empty-anchor.js"></script>





<!--
Font-awesome icons ie github or twitter
-->
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/brands.css" integrity="sha384-n9+6/aSqa9lBidZMRCQHTHKJscPq6NW4pCQBiMmHdUCvPN8ZOg2zJJTkC7WIezWv" crossorigin="anonymous">

<!--
Google fonts api stuff
-->
<link href='https://fonts.googleapis.com/css?family=Rasa' rel='stylesheet'>
<link href='https://fonts.googleapis.com/css?family=Rasa' rel='stylesheet'>

<!--
Here are the required style attributes for css to make this poster work :)
-->
<style>
@page {
size: 33.1in 46.8in;
margin: 0;
padding: 0;
}
body {
margin: 0px;
padding: 0px;
width: 33.1in;
height: 46.8in;
text-align: justify;
font-size: 45px;
line-height: 1.05;
}
/* RMarkdown Class Styles */
/* center align leaflet map,
from https://stackoverflow.com/questions/52112119/center-leaflet-in-a-rmarkdown-document */
.html-widget {
margin: auto;
position: sticky;
margin-top: 2cm;
margin-bottom: 2cm;
}
.leaflet.html-widget.html-widget-static-bound.leaflet-container.leaflet-touch.leaflet-fade-anim.leaflet-grab.leaflet-touch-drag.leaflet-touch-zoom {
position: sticky;
width: 100%;
}
pre.sourceCode.r, pre.sourceCode.python {
background-color: #ffffff20;
border: solid #008080 1mm;
border-radius: 0.5em;
padding: 4mm;
margin: auto;
margin-top: 1em;
margin-bottom: 1em;
/* align-items: center; */
}
div.sourceCode {
background-color: transparent;
width: 85%;
margin-left: auto;
margin-right: auto;
}
code.sourceCode.r, code.sourceCode.python {
background-color: transparent;
font-size: 20pt;
border-radius: 2mm;
}
code {
font-size: 25pt;
font-family: monospace;
background-color: #00808024;
color: #008080;
padding: 1.2mm;
line-height: 1;
border-radius: 2mm;
}
caption {
margin-bottom: 10px;
font-size: 20pt;
font-style: italic;
}

tbody tr:nth-child(odd) {
    background-color: #00808020;
}
.table>thead>tr>th, .table>tbody>tr>th, .table>tfoot>tr>th, .table>thead>tr>td, .table>tbody>tr>td, .table>tfoot>tr>td{
  border-spacing: 0;
  font-size: 40%;
  border-style: none;
  padding-top: 15px;
  padding-bottom: 15px;
  padding-right: 1em;
  padding-left: 1em;
  line-height: 1em;
}
table {
  margin: auto;
}
th {
  padding-left: 5mm;
  padding-right: 5mm;
}
.caption {
font-size: 20pt;
font-style: italic;
padding-top: 0;
}
.references {
font-size: 20px;
line-height: 90%;
}
/* Create three unequal columns that floats next to each other */
.column {
float: left;
padding: 0px;
}
.outer {
width: 33.1in;
height: calc(46.8in *  (1 - 0.2 - 0.1 - 0.01) );
-webkit-column-count: 3; /* Chrome, Safari, Opera */
-moz-column-count: 3; /* Firefox */
column-count: 3;
-webkit-column-fill: auto;
-moz-column-fill: auto;
column-fill: auto;
column-gap: 0;
padding-left: 0cm;
padding-right: 0cm;
/* -webkit-column-rule-width: 50%;
-moz-column-rule-width: 50%;
column-rule-width: 50%; */
-webkit-column-rule-style: none;
-moz-column-rule-style: none;
column-rule-style: none;
-webkit-column-rule-color: black;
-moz-column-rule-color: black;
column-rule-color: black;
background-color: #ffffff;
font-family: Rasa;
margin-top: calc(46.8in *  0.2 );
padding-top: 1em;
padding-bottom: 1em;
}
span.citation {
  color: #008080;
  font-weight: bold;
}
a {
text-decoration: none;
color: #008080;
}
#title {
font-size: 125pt;
text-align: left;
margin: 0;
line-height: 98%;
border-bottom: 0;
font-weight: normal;
background: 0;
}
#author {
color:#0b4545;
margin: 0;
line-height: 85%;
font-size: 1.17em;
}
#affiliation {
padding-top: 0.1em;
color: ;
font-style: italic;
font-size: 25px;
margin: 0;
}
sup {
color: #cc0000;
}
.affiliation sup {
font-size: 20px;
}
.author {
text-align: left;
}
.author sup {
font-size: 30px;
}
.author_extra {
color: #008080;
margin: 0;
line-height: 85%;
font-size: 35px;
text-align: left;
}
.outer h1, h2, h3, h4, h5, h6 {
text-align: center;
margin: 0;
font-weight: bold;
}
.section h1 {
  text-align:center;
  padding-bottom:5px;
  background:
    linear-gradient(
      to left,
      #ffffff 1%,
      #ffffff 20%,
      #0b454575 33%,
      #0b4545 50%,
      #0b454575 66%,
      #ffffff 80%,
      #ffffff 99%
    )
    left
    bottom
    #ffffff
    no-repeat;
  background-size:100% 5px ;
  margin-top: 0.5em;
  margin-bottom: 0.5em;
}
.outer h2 {
text-align: center;
}
.outer p, .level2 {
color: #000000;
}
.outer ol {
padding-left: 8%;
padding-right: 8%;
text-align: left;
}
.main {
width: 33.1in;
height: calc(46.8in * 0.2);
position: absolute;
background-color: #0b4545;
color: #ffffff90;
font-family: Rasa;
background-image: linear-gradient(#0b4545 50%,#008080);
}
.main strong {
color: #ffffff;
}
.main strong > sup {
color: #ffffff;
}
.main sup {
color: #ffffff90;
}
#main-img-left {
width: 10%;
left: 0.5in;
bottom: 0.2in;
position: absolute;
}
#main-img-center {
width: 10%;
left: calc(33.1in * 0.45);
bottom: 0.5in;
position: absolute;
}
#main-img-right {
width: 10%;
right: 0.5in;
bottom: 0.2in;
position: absolute;
}
.main p {
font-size: 150px;
font-family: Rasa;
text-align: center;
margin: 0;
position: absolute;
top: 50%;
-ms-transform: translateY(-50%);
transform: translateY(-50%);
margin-left: 1em;
}
.fab {
color: #00000060;
font-size: 25px;
}
.twitter, i {
color: #00000060;
font-size: 35px;
text-decoration: none;
}
a.email {
text-decoration: none;
color: #00000060;
font-size: 35px;
}
.envelope {
color: #00000060;
font-size: 5px;
text-decoration: none;
}
.globe {
color: #00000060;
font-size: 5px;
text-decoration: none;
}
.globetext {
color: #00000060;
font-size: 35px;
text-decoration: none;
}
.poster_wrap {
width: 33.1in;
height: 46.8in;
padding: 0cm;
}
.main_bottom {
width: 33.1in;
height: calc(46.8in * 0.1);
margin-top: calc(46.8in * (1 - 0.1));
position: absolute;
background-color: #0b4545;
background-image: linear-gradient(#008080 10%, #0b4545);
}
.section {
  padding-left: 10mm;
  padding-right: 10mm;
}
span > #tab:mytable {
  font-weight: bold;
}
.orcid img {
  width: 3%;
}
.emphasis {
  background-color: #008080;
  color: #ffffff;
  border: solid #0b2045 3mm;
  margin: 1em;
  padding-left: 0;
  padding-right: 0;
}
.emphasis h1 {
  font-weight: bold;
  background: none;
  background-color: #0b2045;
  padding-bottom: 5mm;
  padding-top: 1mm;
  margin-top: -1mm;
  margin-right: -1mm;
  margin-left: -1mm;
}
.emphasis blockquote {
  border: 0;
}
.emphasis ol {
  padding: 0;
  padding-left: 8%;
  font-size: 100%;
  font-weight: bold;
}
.emphasis p {
  color: #ffffff;
}
</style>
</head>
<body>


<div class="poster_wrap">

<div class="column outer">
<div class="section">
<h1 id="title">Aquatic Plant eDNA with Nanopore</h1><br>
<h3 id="author" class="author">

Rob Harbert<sup> 1, 2, <a class="orcid" href="https://orcid.org/0000-0002-1714-5033"><img src="https://raw.githubusercontent.com/brentthorne/posterdown/master/images/orcid.jpg"></a></sup><br>
<a class="twitter" href="https://mobile.twitter.com/robharbert"><i class="fab fa-twitter"></i>&nbsp;&nbsp;@robharbert</a><br>

<a class='envelope'><i class="fas fa-envelope"></i></a> <a href="mailto:rharbert@stonehill.edu" class="email">rharbert@stonehill.edu</a> <br>
</h3>

<h5 id="author_extra", class="author_extra">
</h5>


<p id="affiliation" class="affiliation">
<sup>1</sup> Department of Biology, Stonehill College<br> <sup>2</sup> Sackler Institute for Comparative Genomics, American Museum of Natural History
</p>
</div>

<style>
p.caption {
 margin-top: 0;
 margin-bottom: 0;
}
</style>
<div id="premise-of-the-study" class="section level1">
<h1><em>Premise of the study</em></h1>
<p>Cellular and cell-free DNA obtained from environmental samples of soil, water, and air can detect local biodiversity. Environmental DNA (eDNA) sequencing typically relies on standard molecular techniques and resources. Here we present a fully portable workflow designed to work with limited infrastructure for the collection, isolation, extraction, sequencing, and analysis of aquatic eDNA for the monitoring of plant biodiversity.</p>
</div>
<div id="methods" class="section level1">
<h1><em>Methods</em></h1>
<p>We extracted aquatic eDNA from an open pond and a seasonal wetland. Samples were amplified targeting 350bp to 890bp segments from <em>psbA3</em>, <em>rbcLa</em>, <em>ITS2</em>, <em>18S</em>, <em>trnL</em> genomic regions. Amplified DNA was barcoded, pooled up to twelve reactions per run, and sequenced on the Oxford Nanopore MinION sequencer using the low-throughput Flongle flowcells. Amplicon sequences were classified using Kraken2 and BLAST followed by a lowest common ancestor algorithm using the NCBI taxonomy. Table <a href="#tab:flowtable">1</a>.</p>
<table>
<caption>
<span id="tab:flowtable">Table 1: </span>General Workflow
</caption>
<thead>
<tr>
<th style="text-align:left;">
Step
</th>
<th style="text-align:left;">
Description
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Sample Collection
</td>
<td style="text-align:left;">
Field water filtration with 0.045 μm pore filter
</td>
</tr>
<tr>
<td style="text-align:left;">
DNA Extraction
</td>
<td style="text-align:left;">
DNA Extraction with DNEasy PowerWater Kit
</td>
</tr>
<tr>
<td style="text-align:left;">
Target Amplification
</td>
<td style="text-align:left;">
PCR Target Amplification and 12x Multiplex Barcoding
</td>
</tr>
<tr>
<td style="text-align:left;">
Sequencing
</td>
<td style="text-align:left;">
PCR Barcoding kit (SQK-PBK004) library preparation
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Equimolar Pooled Sequencing
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Flongle R9.4.1 flowcells on MinION sequencer
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioinformatics
</td>
<td style="text-align:left;">
Basecalling with Guppy 3.4.5 + high accuracy ‘flip-flop’ model
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Kraken2 classification
</td>
</tr>
</tbody>
</table>
<p>The full protocol is available here: <a href="https://dx.doi.org/10.17504/protocols.io.bc4wiyxe">dx.doi.org/10.17504/protocols.io.bc4wiyxe</a></p>
<div class="figure" style="text-align: center"><span id="fig:classifig"></span>
<img src="lc_port_2021_files/figure-html/classifig-1.png" alt="eDNA Classification Results" width="100%" />
<p class="caption">
Figure 1: eDNA Classification Results
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:ordifig"></span>
<img src="lc_port_2021_files/figure-html/ordifig-1.png" alt="Ordination plots" width="100%" />
<p class="caption">
Figure 2: Ordination plots
</p>
</div>
</div>
<div id="results" class="section level1">
<h1><em>Results</em></h1>
<p>Sequence analysis reveals highly local differences in plant diversity including both aquatic and terrestrial plant taxa Figure <a href="#fig:classifig">1</a> and Figure <a href="#fig:ordifig">2</a>.</p>
<p>The workflow built around this platform provides end-to-end amplicon-based sequencing that can be used to detect nearby plant diversity. The Oxford Nanopore MinION sequencer and the low throughput Flongle flowcell provides a sequencing platform that combines portability, low cost, and ease of use that are essential features for field applications of amplicon metabarcoding.</p>
</div>
<div id="next-steps" class="section level1">
<h1><em>Next Steps</em></h1>
<ul>
<li>Experiment with primer pools (increase multiplexing)</li>
<li>Exchange rapid for ligation chemistry in library prep</li>
<li>Pipeline publication: <a href="https://github.com/developing-bioinformatics/nano_edna" class="uri">https://github.com/developing-bioinformatics/nano_edna</a></li>
<li>Bonito Basecalling -&gt; Higher single molecule accuracy will improve taxonomic classification</li>
<li>Expand to 96 barcodes and MinION flowcells</li>
</ul>
</div>
<div id="resources" class="section level1">
<h1><em>Resources</em></h1>
<ul>
<li><a href="https://github.com/developing-bioinformatics/eDNA_BLAST">Code Repository</a></li>
<li><a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA488629">NCBI BioProject: PRJNA488629</a></li>
<li><a href="https://developing-bioinformatics.github.io/nano_edna/lc_2021/lc_2021.html">Live Version of this Poster <em>check here for updates</em></a></li>
</ul>
</div>

</div>
<div class="main">
<p>Monitoring plant biodiversity in aquatic environmental DNA with low-cost Nanopore Flongle sequencing.</p>
</div>
<div class="main_bottom">
<img id="main-img-left" src=https://www.artstor.org/wp-content/uploads/2013/12/logo_amnh-768x384.jpg>
<img id="main-img-center" src=>
<img id="main-img-right" src=https://www.logolynx.com/images/logolynx/36/36c6948e4b3a8d4237427359aa4f04e1.jpeg>
</div>
</div>



</body>
</html>
